physics/mp_thompson.F90

```diff
@@ -304,7 +304,7 @@ subroutine mp_thompson_init(ncol, nlev, con_g, con_rd, restart,   &
 
            ! If qc is in boundary conditions but nc is not, calculate nc from qc, rho and nwfa
            if (maxval(qc_mp)>0.0 .and. maxval(nc_mp)==0.0) then
-             nc_mp = make_DropletNumber(qc_mp*rho, nwfa) * orho
+             nc_mp = make_DropletNumber(qc_mp*rho, nwfa*rho) * orho
```

Comment:
This should already be a part of PR#567

Reply:
If it is, I'll be giving a merge conflict to resolve, so no problem.